plugin-server/src/config/kafka-topics.ts

```diff
@@ -37,3 +37,6 @@ export const KAFKA_CLICKHOUSE_SESSION_RECORDING_EVENTS = `${prefix}clickhouse_se
 export const KAFKA_CLICKHOUSE_SESSION_REPLAY_EVENTS = `${prefix}clickhouse_session_replay_events${suffix}`
 // write performance events to ClickHouse
 export const KAFKA_PERFORMANCE_EVENTS = `${prefix}clickhouse_performance_events${suffix}`
+
+// log entries for ingestion into clickhouse
+export const KAFKA_LOG_ENTRIES = `${prefix}log_entries${suffix}`
```

Comment:
Would it be worth calling this `console_log_entries` or something to denote what it is? Conscious we have `plugin_log_entries` above

Reply:
Tomas added a `log_entries` table so we can store any log in it... Batch export logs are already going there (otherwise normally I'd agree more specific is better :))